graphql/schema.graphql

```diff
@@ -203,6 +203,16 @@ type Mutation {
     @withPermissions
     @checkUserAccess
     @cacheControl(scope: PRIVATE)
+  createUser(
```

Comment:
I understand the idea of creating a new mutation, but I worry because it'll be very confusing to have both `addUser` and `createUser`. The names don't make it clear when to use one or the other, or any differences in behavior. 

The main difference seems to be the enforced admin access. Why exactly does this `createUser` need this special permission check? Is the behavior very different, in a way that we should restrict this use case to admin users?

If so, maybe call this one `createUserFromAdmin`, so that this is clearer for the caller. It would also be helpful to add a comment (or comments, one per mutation) explaining the main differences between them.

Reply:
We recently added the following mutations for the bulk-import use case:
```
  createOrganizationAndCostCentersWithId(
    input: OrganizationInput!
  ): MasterDataResponse
    @checkAdminAccess
    @cacheControl(scope: PRIVATE)

  createCostCenterWithId(
    organizationId: ID
    input: CostCenterInput!
  ): MasterDataResponse
    @checkAdminAccess
    @cacheControl(scope: PRIVATE)
```

As you can see, both have the same permission check. As we are adding the new `createUser` for the same bulk-import use case, I wanted to keep the same permission check as these other 2 mutations. I didn't want to change the permissions for the `addUser` mutation to avoid breaking it.
On top of that, this `createUser` mutation is cleaner than the other as it expects less parameters, explicitly expects the required fields (using `ID!`) and does only the necessary permission checks (while `addUser` does additional user/roles permission checks before creating the user).

I agree it is not great to have 2 mutations with very similar names like this, so let me think of a better name for it. I can also add a comment on top of each function just to clarify when to use each one.